docker/Dockerfile.base.tmpl

```diff
@@ -23,6 +23,8 @@ RUN --mount=type=cache,target=/var/cache/apt \
         lsb-release \
         mplayer \
         net-tools \
+        nodejs \
```

Comment:
This should be a multi stage build. No reason to keep this around after build.

Reply:
@vpetersson, in the future, #2060 can help us make changes (even the more sophisticated ones) &mdash; like putting multi-stage builds in templates &mdash; more easily.